drivers/include/net/netdev.h

```diff
@@ -250,6 +250,7 @@ typedef enum {
 struct netdev_radio_rx_info {
     int16_t rssi;       /**< RSSI of a received packet in dBm */
     uint8_t lqi;        /**< LQI of a received packet */
+    int8_t snr;         /**< SNR of a received packet in dBm */
```

Comment:
There is misplaced struct in [`drivers/sx127x/include/sx127x_netdev.h`](https://github.com/RIOT-OS/RIOT/blob/cebea3a23ed8ea101912e5133f6e73b4634fb46c/drivers/sx127x/include/sx127x_netdev.h) that you could move to a more generic position (e.g. `drivers/include/net/netdev/lora.h`) instead of changing the API here.

Reply:
Also note https://github.com/RIOT-OS/RIOT/pull/6797#discussion_r260303912

Reply:
agreed!

Reply:
solved with #11144